server/service/src/sync/translation_and_integration.rs

```diff
@@ -209,13 +209,13 @@ impl PullUpsertRecord {
             }
             StorePreference(record) => StorePreferenceRowRepository::new(con).upsert_one(record),
             Barcode(record) => BarcodeRowRepository::new(con).sync_upsert_one(record),
-            Clinician(record) => ClinicianRowRepository::new(con).upsert_one(record),
+            Clinician(record) => ClinicianRowRepository::new(con).sync_upsert_one(record),
             ClinicianStoreJoin(record) => {
-                ClinicianStoreJoinRowRepository::new(con).upsert_one(record)
+                ClinicianStoreJoinRowRepository::new(con).sync_upsert_one(record)
             }
             FormSchema(record) => FormSchemaRowRepository::new(con).upsert_one(record),
             DocumentRegistry(record) => DocumentRegistryRowRepository::new(con).upsert_one(record),
-            Document(record) => upsert_document(con, record, false),
```

Comment:
@clemens-msupply with 'false' here does it mean document that arrived via sync (form say another site), would be sent back ?